pkg/tcpip/stack/nic.go

```diff
@@ -526,7 +526,8 @@ func (n *NIC) DeliverNetworkPacket(linkEP LinkEndpoint, remote, _ tcpip.LinkAddr
 		} else {
 			// n doesn't have a destination endpoint.
 			// Send the packet out of n.
-			hdr := buffer.NewPrependableFromView(vv.First())
+			// If we want to send the packet to a link-layer, we have to reserve space for an Ethernet header.
```

Comment:
Please wrap comments at 80 columns. (Note the other comments in this file.)

Here and elsewhere in this PR.

Reply:
fixed